src/docs/sphinx/Actions/Pipelines.rst

```diff
@@ -294,5 +318,147 @@ Only the area outside of the implicit function remains.
+  conduit::Node pipelines;
+  // pipeline 1
+  pipelines["pl1/f1/type"] = "clip";
+  // filter knobs
+  conduit::Node &clip_params = pipelines["pl1/f1/params"];
+  clip_params["topology"] = "mesh";
+  clip_params["box/min/x"] = 0.;
+  clip_params["box/min/y"] = 0.;
+  clip_params["box/min/z"] = 0.;
+  clip_params["box/max/x"] = 10.01; // <=
+  clip_params["box/max/y"] = 10.01;
+  clip_params["box/max/z"] = 10.01;
+  
+.. _clipboxfig:
+
+..  figure:: ../images/box_clip.png
+    :scale: 50 % 
+    :align: center
+
+    A box clip of the same data set that removes the octant on the positive x, y, and z axes.
+
+.. code-block:: c++
+
+  conduit::Node pipelines;
+  // pipeline 1
+  pipelines["pl1/f1/type"] = "clip";
+  // filter knobs
+  conduit::Node &clip_params = pipelines["pl1/f1/params"];
+  clip_params["topology"] = "mesh";
+  clip_params["plane/point/x"] = 0.;
+  clip_params["plane/point/y"] = 0.;
+  clip_params["plane/point/z"] = 0.;
+  clip_params["plane/normal/x"] = 1.; 
+  clip_params["plane/normal/y"] = 0.;
+  clip_params["plane/normal/z"] = 0;
+
+.. _clipplanefig:
+
+..  figure:: ../images/clip_plane.png
+    :scale: 50 % 
+    :align: center
+    
+    Clipping by a plane defined by a point on the plane and the plane normal.
+
+:numref:`Figures %s <clipspherefig>`, 
+:numref:`%s <clipsphereInvertedfig>`,  
+:numref:`%s <clipboxfig>`, and  
+:numref:`%s <clipplanefig>` show an images produced from the clip filter. 
+All of the clip  examples are located in the file `clip test <https://github.com/Alpine-DAV/ascent/blob/develop/src/tests/ascent/t_ascent_clip.cpp>`_.
+
+Clip By Field
+~~~~~~~~~~~~~
+The clip by field filter removes cells from the specified topology using the values in a scalar field.
+By default, all values below the clip value are removed from the data set. As with clip by implicit function,
+ the clip can be inverted.
+
+.. code-block:: c++
+
+  conduit::Node pipelines;
+  // pipeline 1
+  pipelines["pl1/f1/type"] = "clip_with_field";
+  // filter knobs
+  conduit::Node &clip_params = pipelines["pl1/f1/params"];
+  clip_params["field"] = "braid";
+  clip_params["clip_value"] = 0.;
+  
+.. _clipfieldfig:
+
+..  figure:: ../images/clip_field.png
+    :scale: 50 % 
+    :align: center
+
+    An example of clipping all values below 0 in a data set. 
+
+.. code-block:: c++
+
+  conduit::Node pipelines;
+  // pipeline 1
+  pipelines["pl1/f1/type"] = "clip_with_field";
+  // filter knobs
+  conduit::Node &clip_params = pipelines["pl1/f1/params"];
+  clip_params["field"] = "braid";
+  clip_params["invert"] = "true";
+  clip_params["clip_value"] = 0.;
+  
+.. _clipfieldinvertedfig:
+
+..  figure:: ../images/clip_field_inverted.png
+    :scale: 50 % 
+    :align: center
+
+    An example of clipping all values above 0 in a data set. 
+
+IsoVolume
+~~~~~~~~~
+IsoVolume is a convenience filter based on clipping a data set based on a minimum 
+and maximum values of a scalar field. All value outside of the minminum and maximum 
```

Comment:
I guess I wouldn't call it a convenience filter, sure its more convenient than setting the the clip by field stuff manually to create an iso volume, but in many ways its a core feature.